llms/mlx_lm/tuner/utils.py

```diff
@@ -0,0 +1,22 @@
+import mlx.core as mx
+from mlx.utils import tree_unflatten
+
+from .linear import LoRALinear
+
+
+def apply_lora_layers(model, adapter_file: str):
```

Comment:
With this helper, we can recover the lora layers and update weights from a single adapter file.